XenCenterLib/EncryptionUtils.cs

```diff
@@ -66,7 +66,7 @@ public static string Protect(string data)
         {
             byte[] dataBytes = Encoding.Unicode.GetBytes(data);
 
-            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.CurrentUser);
+            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.LocalMachine);

```

Comment:
Yes, without the scope change, we cannot read the config from a service. because service runing in system account.